app/components/ColorInput/ColorInput.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function ColorInput({
+  colorLetter,
+  currentNewColor,
+  defaultText,
+  onChange,
+}) {
+  const inputProps = {
+    placeholder: `Default: ${defaultText}`,
+    value: currentNewColor,
+    type: 'text',
+    id: `newColor${colorLetter}`,
+    onChange,
+  };
+  return (
+    <div className="inputWrapper">
+      <label htmlFor={ `newColor${colorLetter}` }>{`color ${colorLetter}:`}
```

Comment:
You don't need an `id` on an `input` and a `for` attribute on a label if the former is nested in the latter. Like here :)
It just works and that's the only way I write input labels, because it avoids dealing with ids and their uniqueness constraint.

Reply:
It's the ESLint...

looking for the rule I need to bend to my will so it doesn't give me the error here.

done